probe/endpoint/nat.go

```diff
@@ -50,7 +50,10 @@ func toMapping(f Flow) *endpointMapping {
 // ApplyNAT duplicates Nodes in the endpoint topology of a
 // report, based on the NAT table as returns by natTable.
 func (n NATMapper) ApplyNAT(rpt report.Report, scope string) {
-	n.WalkFlows(func(f Flow) {
+	if n.Conntracker == nil {
+		return
+	}
```

Comment:
I would like to merge this change, so that my tests no longer segfault. If you're not :+1:, I'll switch my attention to refactoring this package so that these nil conditions don't occur. Please let me know.